### PR TITLE
fucking with the homepage dick

### PR DESCRIPTION
**Pull request by austingavlak**

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 		<div class="row flex-items-xs-top">       <div class="col-xs">&nbsp;</div>
 		<div class="col-xs-9 col-md-6  col-lg-5 boob">         <h1>           Find
-			Holistic Healing Near You.         </h1>         <h3>           Your holistic
+			Holistic Healing Dick Near You.         </h1>         <h3>           Your holistic
 			hub of ancient wisdoms and natural remedies.         </h3>         <div class
```
Comment (austingavlak) [inline app/views/pages/home.html.erb:10]: What you think @beschloo ?!

Comment (austingavlak) [reply app/views/pages/home.html.erb:10]: PRETTY TITE RITE?!

**Comment by sebbean** [comment]
@austingavlak IDK BOUT THIS ONE BRO

**Comment by sebbean** [comment]
screen shot 2016-09-13 at 11 24 27 am
KNAMEEN?

**Pull request merged**